.github/actions/paths-filter/action.yml

```diff
@@ -223,6 +232,14 @@ runs:
           - 'zeebe/gateway-protocol/vacuum-ruleset.yaml'
           - 'zeebe/gateway-protocol/vacuum-ignores.yaml'
 
+        rdbms-change:
+        - 'db/rdbms/**'
+        - 'db/rdbms-schema/**'
+        - 'parent/**'
```

Comment:
Redundant if output variable uses `steps.filter-common.outputs.maven-change == 'true' ||` (which is more complete).

Reply:
I agree that ` parent/**` is redundant when using `maven-change` plus `rdbms-change`. For the rest I think that they are not redundant, since ` mvn-changes` doesn't detect changes to source code, config files, or schemas. So although `maven-change` may seem more broadly scoped for Maven-related changes, it seems to me, we still need `rdbms-change` for detecting functional or test changes related to RDBMS. Does it make sense?

Reply:
Correct, I should've made it more clear that my comment was only referring to the `parent/**` line.